src/index.html

```diff
@@ -1,12 +1,334 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="catalog">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="catalog__body">
+
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="./images/logo.png" alt="moyo-logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link
+              nav__link--active
+               is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+             href="#laptops&computers"
+             class="nav__link"
+             data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class=" nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog__cards-container">
+      <article class="card" data-qa="card">
+        <img src="./images/imac.jpeg" class="card__image"
+          alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)">
```

Comment:
```suggestion
        <img 
          src="./images/imac.jpeg" 
          class="card__image"
          alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
        >
```
keep proper tags and attributes formatting

Reply:
check everywhere